### ISSUE TITLE
Love the mod but not option to disable teleporting?

### ISSUE THREAD

**Issue by lumagatto** [gitsh01/libertyvillagers#120]
Is there a reason, for example performance?

**Comment by burghenby** [comment]
I was also wondering why why there was no option to disable the teleport? Or if there is an option to disable it where is it? I regularly relocate villagers to new homes for one reason for another and build walls around their villages. Them having the ability to teleport out of the confines of their room before they settle and accept their new home, or teleport out of the borders of their village defeats the purpose of my building the wall after fixing up their village in the first place.

**Comment by mauritson-dev** [comment]
Teleport will break some villager breeders, as baby villagers can teleport to a nearby bed they cannot path find to normally.  Also, cycling through professions by breaking a workstation, a villager will teleport out of an enclosed space to a nearby workstation.  Please allow this feature to be toggleable (default: OFF).  Thanks,

**Comment by iMoobert** [comment]
I also have an issue where villagers are just appearing outside my enclosed village's walls. There should be no way of them getting out, but when there are no openings in the wall, I slowly bleed villagers for some reason. They just appear outside the wall, disconnect from their workstation/bed, wander, then die.

**Comment by z1haze** [comment]
yes, please allow us to disable this unwanted "pathfinding" that results in villagers teleporting out of their area.
